docs/source/pages/howToWorkWithPlatform/mainApplication/envelope/approvalProcess/approved.rst

```diff
@@ -60,3 +60,57 @@ How to send envelope for approval
 
 11. When approval flow process is finished (all approvers agreed, someone from approvers reject), envelope returns to status "Waiting". User, that started approval can process envelope further or start new approval process.
 
+
+
+
+.. toctree::
+
+
+How to use approval   flow
+==========================
+
+1.User can send envelope for approval in the “Waiting” status or a draft envelope. To do this open envelope in “Waiting” status, in the upper right corner of the envelope header click on [Approval] button.
+
+2.Select  New route
+
+.. image:: pic_flow/Screenshot_11.png
+   :width: 600
+   :align: center
+
+3.In the input “Аpprover” and “Mailbox” (you can search mailbox by name or uuid) enter details of the approver and click on [Add] button. You can create unlimited quantity of approvers. You can delete approver by clicking on [Cancel] button.You can mark envelope with "Urgent" status(1), and you can et order for approvers(2).
+
+.. image:: pic_flow/Screenshot_2.png
+     :width: 600
+     :align: center
+
+4.As soon as you finish approval flow creation you can click on [Confirm] button to start approval process.
+
+.. image:: pic_flow/Screenshot_3.png
+     :width: 600
+     :align: center
+
+5.You can save the route.
+
+.. image:: pic_flow/Screenshot_4.png
+     :width: 600
+     :align: center
+
+6.To save route, input name for route and click [Save] button.
+
+.. image:: pic_flow/Screenshot_5.png
+     :width: 600
+     :align: center
+
+7.You can use saved routes.
```

Comment:
You can use saved routes. To do this on the first step of approval process select option "Use existing route"